std_msgs/Header header
bool being_idle